src/Console/Command/RunCommand.php

```diff
@@ -83,4 +93,25 @@ protected function paths(): PathsHelper
     {
         return $this->getHelper(PathsHelper::HELPER_NAME);
     }
+
+    /**
+     * Split $value on ",", trim the individual parts and
+     * de-deduplicate the remaining values
+     *
+     * @param string $value
+     * @return string[]
+     */
+    protected function parseCommaSeparatedOption(string $value)
```

Comment:
It is better to move this method to the `\GrumPHP\Util\Str` class.
Maybe something like:
```
public static function explode(string $delimiter, string $string): array
```

This way the logic can be tested in a normal way and it is reusable if we want to add more comma separated options?